ocs_ci/ocs/ui/base_ui.py

```diff
@@ -307,6 +309,27 @@ def wait_until_expected_text_is_found(self, locator, expected_text, timeout=60):
             )
             return False
 
+    def check_element_presence(self, locator, timeout=5):
+        """
+        Check if an web element is present on the web console or not.
+
+        Args:
+             locator (tuple): (GUI element needs to operate on (str), type (By))
+             timeout (int): Looks for a web element repeatedly until timeout (sec) occurs
+        return:
+            bool: True if the element is found, raises NoSuchElementException and returns False otherwise
```

Comment:
```python
"""
    bool: True if the element is found 

Raises:
    NoSuchElementException and returns False otherwise

"""
```

Reply:
The function does not raise any exception. NoSuchElementException is being caught and the function returns False